src/components/ContactCard/ContactFieldInput.jsx

```diff
@@ -0,0 +1,93 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Field } from "react-final-form";
+import classnames from "classnames";
+
+const getInputComponent = inputType =>
+  inputType === "textarea" ? "textarea" : "input";
+
+class ContactFieldInput extends React.Component {
+  state = {
+    renderLabel: false,
```

Comment:
As a boolean, this variable sould be named with a prefix like `is` or `has`, should not it?